lib/hr/src/main.rs

```diff
@@ -16,7 +16,7 @@ fn main() -> Result<()> {
 }
 
 async fn run() -> Result<()> {
-    let listen_addr = env::var("LISTEN_ADDR").unwrap_or_else(|_| "localhost:8000".to_owned());
+    let listen_addr = env::var("PORT").unwrap_or_else(|_| "0.0.0.0:3000".to_owned());
```

Comment:
I would expect PORT to contain an integer, but if I set it to an integer then I get:
```
$ (cd lib/hr/ && PORT=3000 cargo run --quiet)
Playground: http://3000
Error: Custom { kind: InvalidInput, error: "unable to parse listener from `3000`" }
```

(I'm happy for it to stay as LISTEN_ADDR or be changed to accept an integer)

Reply:
I've changed this to LISTEN_HOST_AND_PORT, which I think is the most accurate description